src/js/control-bar/text-track-controls/text-track-button.js

```diff
@@ -62,4 +62,4 @@ class TextTrackButton extends MenuButton {
 }
 
 MenuButton.registerComponent('TextTrackButton', TextTrackButton);
-export default TextTrackButton;
```

Comment:
Oh! I haven't removed the line, only the EOF empty line (stupid Atom doing that all the time :/). You can check it's still there.